src/mame/drivers/magicard.cpp

```diff
@@ -1010,14 +839,15 @@ void magicard_base_state::magicard_base(machine_config &config)
 	SCC68070(config, m_maincpu, CLOCK_C); /* SCC-68070 CCA84 */
 
 	SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
-	
+
```

Comment:
I think you want to set the update per scanline flag here – you want to set either:
```
m_screen->set_video_attributes(VIDEO_UPDATE_SCANLINE);
```
or
```
m_screen->set_video_attributes(VIDEO_UPDATE_SCANLINE | VIDEO_UPDATE_ALWAYS);
```
depending on whether the scanline update has persistent side effects.

Reply:
I'd already got m_screen->set_video_attributes(VIDEO_UPDATE_SCANLINE); but it was further down in the function so easy to miss.
I've reorganised the magicard_base() function to group all the config of each device together to make it clearer what's happening.